src/systems/geometry.js

```diff
@@ -59,9 +59,13 @@ module.exports.System = registerSystem('geometry', {
     var cache = this.cache;
     var cacheCount = this.cacheCount;
     var geometry;
-    var hash = this.hash(data);
+    var hash;
+
+    if (data.skipCache || data.mergeTo) { return; }

```

Comment:
Doesn't seem like you need this because you already check for `skipCache` in `geometry.mergeTo`